quarkus-qe/quarkus-startstop
# Quarkus 1.7 continuation

Quarkus 1.7 related updates - continuation

- JWT doesn't need fake configuration anymore
- SpecialCharsTest - Honor Quarkus version in dev mode
- README - pointer to the details about CodeQuarkusExtensions enum
- Update of whitelist for Quarkus 1.7
- Update of CodeQuarkusExtensions enum for Quarkus 1.7 

There are 2 fails expected to appear - SpecialCharsTest.diacriticsDEV and SpecialCharsTest.otherDEV
I will file Quarkus issue for it
